x-pack/legacy/plugins/monitoring/public/components/elasticsearch/nodes/nodes.js

```diff
@@ -82,25 +86,18 @@ const getColumns = (showCgroupMetricsElasticsearch, setupMode, clusterUuid) => {
         <div>
           <div className="monTableCell__name">
             <EuiText size="m">
-              <EuiToolTip
-                position="bottom"
-                content={node.nodeTypeLabel}
-              >
-                <span className={`fa ${node.nodeTypeClass}`} />
+              <EuiToolTip position="bottom" content={node.nodeTypeLabel}>
+                <EuiIcon type={node.nodeTypeClass} />
```

Comment:
This is now causing a console error because `node.nodeTypeClass` doesn't always exist. I'd recommend wrapping in a truthy statement so EuiIcon only renders if an class is present. Like:
```suggestion
                {node.nodeTypeClass && <EuiIcon type={node.nodeTypeClass} />}
```

<img width="951" alt="Screen Shot 2019-11-27 at 12 27 08 PM" src="https://user-images.githubusercontent.com/549577/69747503-7c42eb80-1114-11ea-84ed-219cc058e025.png">